.vehicle_profiles/buildSupportedVehiclesList.js

```diff
@@ -0,0 +1,14 @@
+import { readFile, writeFile } from 'fs/promises';
+import { glob } from 'glob'
+
+const vehicleProfilesJson = '../vehicle_profiles.json'
+const automateDirectory = await glob('../docs/content/*.Config/*.Automate')
+const target = `${automateDirectory}/2.Supported_Vehicles.md`
```

Comment:
Dosen't glob return an array of files? so I doubt this would work? Unless theres some string interpilation magic happening?

I think I'd rather glob the entire path, `../docs/content/*.Config/*.Automate/*.Supported_Vehicles.md` and then error if more than 1 file is found, to make sure its not going to wipe out some extra/random file

Reply:
Done